xml/xen2kvm.xml

```diff
@@ -1057,10 +1058,9 @@ getty@xvc1.service -&gt; /usr/lib/systemd/system/getty@xvc1.service
 
 </screen>
     <para>
-     Save the configuration to a file in your home directory. After you later
-     import it, it will be copied to the default
-     <filename>/etc/libvirt/qemu</filename>. Suppose you save the file as
-     <filename>SLES11SP3.xml</filename>.
+     Save the configuration to a file in your home directory, for example, as
+     <filename>SLES11SP3.xml</filename>. Later, after you import it, it will be
+     copied to the default <filename>/etc/libvirt/qemu</filename> folder.
     </para>
```

Comment:
```suggestion
     Save the configuration to a file in your home directory, as
     <filename>SLES11SP3.xml</filename>, for example. After you import it, it will be
     copied to the default <filename>/etc/libvirt/qemu</filename> folder.
    </para>
```

Reply:
Move insertion 'for example' to the end for better flow of reading. Remove 'Later' because the timing should be clear by 'after you import it' already.